src/RuleSet/RuleSet.php

```diff
@@ -103,31 +106,31 @@ public function getLineNo(): int
 
     public function addRule(Rule $ruleToAdd, ?Rule $sibling = null): void
     {
-        $sRule = $ruleToAdd->getRule();
-        if (!isset($this->rules[$sRule])) {
-            $this->rules[$sRule] = [];
+        $propertyOrRuleName = $ruleToAdd->getRule();
```

Comment:
Having looked into how this works and what it does, I now think just `$propertyName` would be better - see https://github.com/MyIntervals/PHP-CSS-Parser/pull/1051#pullrequestreview-2652886522.